main/src/ca/uwaterloo/flix/util/Options.scala

```diff
@@ -47,6 +47,7 @@ object Options {
     xfuzzer = false,
     xprinttyper = None,
     xverifyeffects = false,
+    xverifylocations = false,
     xsubeffecting = Set.empty,
     XPerfN = None,
     XPerfFrontend = false,
```

Comment:
And then below here (line 61) set the verify flag to true so its used in the tests